lms/djangoapps/dashboard/tests/test_sysadmin.py

```diff
@@ -290,6 +290,7 @@ def test_gitlog_no_logs(self):
 
         self._rm_edx4edx()
 
+    @unittest.skipIf(settings.SKIP_EDRAAK_UNWANTED_TESTS, 'Could not fix it on CircleCI.')
```

Comment:
Hey, @felipemontoya. Glad to have your input here. I guess this is just a flaky test that is highly environment dependent. I meant it does a git history, which can be vary a lot between one configuration and another depending on whether you shallow copy, have the tags or not and if you're actually using git instead of `cp -r`.

For me it always failed, but I only tested it a couple of times and gave up on it.

I'd not worry about it much, this is just one test that can be easily refactored. I'm not sure if the selenium tests can be run on CircleCI or not, and that's what #3 is about.